test/shared/src/main/scala/zio/test/DefaultTestReporter.scala

```diff
@@ -79,82 +81,9 @@ object DefaultTestReporter {
                 }
             },
             initialDepth * 2,
-            List(annotations), {
```

Comment:
This is a smaller change than it might appear. I just moved this logic to a separate def and then included `summaryLines` as additional output.